docs/rest/DaVinciFormulary/DaVinciFormulary_Sample_Resources.http

```diff
@@ -0,0 +1,2183 @@
+# This file includes sample resources that you can use to test DaVinci Formulary
+### REST Client
+@fhirurl= <FHIR URL>
+@clientid= <CLIENT ID>
+@clientsecret= <CLIENT SECRET>
+@tenantid= <SUBSCRIPTION ID>
+@contentType = application/json
+
+### Get Metadata - Allows you to get the current capability statement
+# this will update as you add search parameters and profiles to the database
+GET {{fhirurl}}/metadata
+
+### Get access token
+# @name getAADToken
+POST https://login.microsoftonline.com/{{tenantid}}/oauth2/token
+Content-Type: application/x-www-form-urlencoded
+
+grant_type=client_credentials
+&resource={{fhirurl}}
+&client_id={{clientid}}
+&client_secret={{clientsecret}}
+
+### 
+GET {{fhirurl}}/StructureDefinition?url=http://hl7.org/fhir/us/davinci-drug-formulary/StructureDefinition/usdf-FormularyDrug
+Authorization: Bearer {{token}}
+content-type: {{contentType}}
+
+### Capture access token from getToken request
+@token={{getAADToken.response.body.access_token}}
+
+
+### Variables for creating resources
+# Add values to each ID as you create them.
+# You need to create in order and add IDs as you go or references will not work
+
+@formularydrugV1002=<formularydrugV1002>
+@FormularyDrugV3001=<FormularyDrugV3001>
+@cmsip9=<cmsip9>
+@CoveragePlanV1002=<CoveragePlanV1002>
+@CoveragePlanV3001=<CoveragePlanV3001>
+@CoveragePlanV3002=<CoveragePlanV3002>
+@CoveragePlanV3004t=<CoveragePlanV3004t>
+@FormularyDrugV3002=<FormularyDrugV3002>
```

Comment:
Instead of the user entering these you can have them be set from the response of the POST requests. If you name the requests you can use lines like this:
```@formularydrugV1002={{formularydrugV1002Request.response.body.id}}```

Reply:
@moiradillon12 - Any chance you could do this on the C4BB one as well?